NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 * Examples now use `scda.2022` instead of `scda.2021`.
 * Transferred data hashing step in `FilteredDataset` to `teal`.
+* Replaced `ReactiveQueue` objects with `shiny::reactiveVal` objects.
```

Comment:
Please don't bother with some internal change. FilterStates and ReactiveQueue were internal objects
```suggestion
```

Reply:
:+1: